doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -0,0 +1,123 @@
+  MbedTLS is used by default.
+
+See the Kconfig help for details.
+
+Override of default Kconfig values
+==================================
+
+To simplify configuration, the GFPS modifies default values of related Kconfig options to follow Fast Pair requirements.
+The service also enables some of the functionalities using Kconfig select statement.
+
+Bluetooth privacy
+-----------------
+
+The service selects Bluetooth privacy (:kconfig:option:`CONFIG_BT_PRIVACY`).
+The service sets the default value of :kconfig:option:`CONFIG_BT_RPA_TIMEOUT` to 3600 seconds.
+During not-discoverable advertising, the RPA address rotation shall be done together with Fast Pair payload update.
+Because of this, the RPA address cannot be rotated by Zephyr in the background.
+
+Firmware Revision characteristic
+--------------------------------
+
+The Fast Pair specification requires enabling GATT Device Information Service and the Firmware Revision characteristic.
+The default values of :kconfig:option:`CONFIG_BT_DIS` and :kconfig:option:`CONFIG_BT_DIS_FW_REV` are set to enabled.
+The default value of :kconfig:option:`CONFIG_BT_DIS_FW_REV_STR` is set to :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` if :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` is enabled.
+Otherwise, it is set to ``0.0.0+0``.
+
+MTU configuration
+-----------------
+
+The Fast Pair specification suggests using ATT maximum transmission unit (MTU) value of ``83`` if possible.
+Because of this requirement, the default values of the following Kconfig options are modified by the GFPS Kconfig:
+
+* :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU`
+* :kconfig:option:`CONFIG_BT_BUF_ACL_TX_SIZE`
+* :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE`
+* :kconfig:option:`CONFIG_BT_CTLR_DATA_LENGTH_MAX`
+
+.. tip::
+   In case of the nRF53, this part of the configuration cannot be automatically updated for the network core and it must be manually aligned by the user.
+   These options on the network core shall be set to default values specified by GFPS Kconfg.
+
+Partition Manager
+-----------------
+
+The Fast Pair provisioning data is preprogrammed to a dedicated flash memory partition.
+The GFPS selects :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` to enable the :ref:`partition_manager`.
+
+Settings
+--------
+
+The GFPS uses Zephyr's :ref:`zephyr:settings_api` to store Account Keys.
+Because of this, the GFPS selects :kconfig:option:`CONFIG_SETTINGS`.
+
+Implementation details
+**********************
+
+The implementation uses :c:macro:`BT_GATT_SERVICE_DEFINE` to statically define and register the Fast Pair GATT service.
+The service automatically handles all of the requests received from the Fast Pair Seeker.
+No application input is required to handle the requests.
+
+Bluetooth authentication
+========================
+
+The service overrides the set of Zephyr's Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`) while handling Key-Based Pairing request.
```

Comment:
Don't this callbacks change in latest upmerge?

Reply:
Yes, the authentication info callback (`bt_conn_auth_info_cb`) were separated. Other callbacks that were part of `bt_conn_auth_cb` are still working in the same way.
I aligned to new callbacks in Fast Pair implementation while rebasing